layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```

Comment:
1. can you please provide more context why is this needed (cant tell from the link) ? 
2. we are including open graph stuff only when `opengraph = true` param is enabled
 (SEE https://github.com/jbub/ghostwriter/blob/master/layouts/partials/header.html#L13)

thanks for your contribution!